src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationNotifications.cs

```diff
@@ -23,7 +23,6 @@ public OpenIdConnectAuthenticationNotifications()
             MessageReceived = notification => Task.FromResult(0);
             SecurityTokenReceived = notification => Task.FromResult(0);
             SecurityTokenValidated = notification => Task.FromResult(0);
-            RedirectToIdentityProvider = notification => Task.FromResult(0);
```

Comment:
This PR is big enough, there's no need to couple it to orthogonal consistency changes, don't change the notification pattern as part of this PR